SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReplicationGraphLoadBalancingHandler.cpp

```diff
@@ -31,6 +31,18 @@ void FSpatialReplicationGraphLoadBalancingContext::AddActorToReplicate(AActor* A
 	AdditionalActorsToReplicate.Add(Actor);
 }
 
+#if ENGINE_MINOR_VERSION > 25
```

Comment:
Minor again, swapping to `>25` here instead of `>= 26`. Might make it easier to find what needs removing in the future with consistency

Reply:
This goes for the whole PR, I think we should choose a style and stick with it